tests/1.8/regression/metadata/fail/stream-undefined-id/metadata

```diff
@@ -0,0 +1,49 @@
+/* CTF 1.8 */
+trace {
+	major = 1;
+	minor = 8;
+	uuid = "ff29057a-fe6b-4996-9d98-06f95dcd55cf";
+	byte_order = le;
+	packet.header := struct {
+		integer { size = 32; align = 8; signed = false; encoding = none; base = decimal; byte_order = le; } magic;
+		integer { size = 8; align = 8; signed = false; encoding = none; base = decimal; byte_order = le; } uuid[16];
+		integer { size = 32; align = 8; signed = false; encoding = none; base = decimal; byte_order = le; } stream_id;
+	} align(8);
```

Comment:
If I remove the packet header, I get the following error from BT:

    [error] ctf_stream_visit: missing stream_id field in packet header declaration, but stream_id attribute is declared for stream.

Seems to work if I keep just the stream_id in the packet header though.